tests/functional/detection_status/test_detection_status.py

```diff
@@ -0,0 +1,148 @@
+#
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+""" detection_status function test. """
+import json
+import os
+import unittest
+
+import shared
+
+from libtest import codechecker
+from libtest import env
+
+
+class TestDetectionStatus(unittest.TestCase):
+
+    def setUp(self):
+        # TEST_WORKSPACE is automatically set by test package __init__.py .
+        self.test_workspace = os.environ['TEST_WORKSPACE']
+
+        test_class = self.__class__.__name__
+        print('Running ' + test_class + ' tests in ' + self.test_workspace)
+
+        self._codechecker_cfg = env.import_codechecker_cfg(self.test_workspace)
+
+        # Get the CodeChecker cmd if needed for the tests.
+        self._codechecker_cmd = env.codechecker_cmd()
+        self._test_dir = os.path.join(os.path.dirname(__file__), 'test_files')
```

Comment:
I would recommend to use the workspace created for the tests to generate test files. Not to the test file directory.